### PR TITLE
Industrial archaeology

### PR DESCRIPTION
**Pull request by FrancescoCioria**

### CODE REVIEW

File: Italy/Lombardia/Archeologia Industriale/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Industrial archaeology in Milan
+
+With this short urban trip you'll have the chance to admire the massive industrial archaeology landmarks of Falck steel industries and Carroponte (ex Breda industries).
+
+We'll also go across a small street full of street art in the NoLo neighborhood, all by cutting through small and big parks.
```
Comment (dhinus) [inline Italy/Lombardia/Archeologia Industriale/README.md]: ❤️

**Review by dhinus — CHANGES_REQUESTED** [review]
Molto bello!

Sistemerei solo tre punticini:

Screenshot 2020-05-18 at 22 58 13

Screenshot 2020-05-18 at 22 58 26

Screenshot 2020-05-18 at 22 57 48

**Review by dhinus — APPROVED** [review]

**Comment by dhinus** [comment]
Se hai voglia, ultime due cosine, altrimenti fai pure merge così:

Screenshot 2020-05-19 at 10 29 40


Screenshot 2020-05-19 at 10 27 50

**Comment by ecamellini** [comment]
❤️ 🏗️

**Review by ecamellini — APPROVED** [review]

**Pull request merged**